.github/workflows/vulnerabilities.yml

```diff
@@ -26,11 +25,11 @@ jobs:
       - name: Restore .NET Dependencies
         run: dotnet restore Sentry.sln --nologo
 
-      # The dotnet list package command doesn't change its exit code on detection, so tee to a file and scan it
+      # The dotnet package list command doesn't change its exit code on detection, so tee to a file and scan it
```

Comment:
note:
- .NET 9 SDK and earlier: `dotnet list package`
- the "noun first" form was introduced in .NET 10 SDK
- see also [dotnet package list](https://learn.microsoft.com/dotnet/core/tools/dotnet-package-list)